packages/edit-widgets/src/test/index.test.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	renderInitialize,
+	screen,
+	getByRole,
+	findByRole,
+	findAllByRole,
+	queryAllByRole,
+	fireEvent,
+} from '@wordpress/testing-library';
+
+/**
+ * Internal dependencies
+ */
+import { initialize } from '..';
+import './mocks/server';
+import availableLegacyWidgets from './mocks/available-legacy-widgets';
+
+// We need this since some tests run over 5 seconds threshold.
```

Comment:
Yeah, only the second test takes that long though. I think it might be an existing performance bug with our component but not a problem with the test itself. I'll try to do some more research to find out why it's so slow.